src/main/java/edu/kis/powp/jobs2d/command/io/dto/CommandDTO.java

```diff
@@ -0,0 +1,21 @@
+package edu.kis.powp.jobs2d.command.io.dto;
+
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+
+public class CommandDTO {
```

Comment:
Could those DTO classes be converted to inner classes in JSON command loader? Having such utility constructs public but without clear API does not seem useful.